tests/integration/helpers/cluster.py

```diff
@@ -1349,9 +1354,24 @@ def create_dir(self, destroy_dir=True):
         instance_config_dir = p.abspath(p.join(self.path, 'configs'))
         os.makedirs(instance_config_dir)
 
+        entrypoint_cmd = CLICKHOUSE_START_COMMAND
+
         print("Copy common default production configuration from {}".format(self.base_config_dir))
-        shutil.copyfile(p.join(self.base_config_dir, 'config.xml'), p.join(instance_config_dir, 'config.xml'))
-        shutil.copyfile(p.join(self.base_config_dir, 'users.xml'), p.join(instance_config_dir, 'users.xml'))
+
+        if self.copy_config_xml:
+            print("Found config.xml configuration: {}".format(self.base_config_dir + '/config.xml'))
+            shutil.copyfile(p.join(self.base_config_dir, 'config.xml'), p.join(instance_config_dir, 'config.xml'))
```

Comment:
Let's call the variables to `config_xml_name` and `users_xml_name` (with default values "config.xml" and "users.xml"), this will allow us to write the copying code only once.